src/client/Microsoft.Identity.Client/AppConfig/BaseAbstractApplicationBuilder.cs

```diff
@@ -73,7 +73,7 @@ public T WithHttpClientFactory(IMsalHttpClientFactory httpClientFactory)
         public T WithHttpClientFactory(IMsalHttpClientFactory httpClientFactory, bool retryOnceOn5xx)
         {
             Config.HttpClientFactory = httpClientFactory;
-            Config.RetryOnServerErrors = retryOnceOn5xx;
+            Config.DisableInternalRetries = retryOnceOn5xx;
```

Comment:
I think this is inversely related. DisableInternalRetries = !retryOnceOn5xx.

Reply:
Great catch. Thanks.